src/nifgen/examples/nifgen_script.py

```diff
@@ -32,15 +71,13 @@ def calculate_rampdown():
 
 def calculate_square():
     time = np.linspace(start=0, stop=10, num=number_of_points)    # np.linspace(start, stop, num=50, endpoint=True, retstep=False, dtype=None)
-    square_build = signal.square(t=time, duty=0.5)              # signal.square(t, duty=0.5)
-    square = square_build.tolist()                              # List of Float
+    square = square_wave(t=time, duty_cycle=0.5)
     return square
 
 
 def calculate_triangle():
```

Comment:
A triangle wave and sawtooth wave are not really the same thing, but I'm going to let it go because I don't want to change example behavior as part of this PR.